ra_aid/agent_utils.py

```
@@ -222,14 +398,15 @@ def run_research_agent(
                 memory=memory,
                 config=config,
                 thread_id=thread_id,
-                console_message=console_message
+                console_message=console_message,
             )
     except (KeyboardInterrupt, AgentInterrupt):
         raise
     except Exception as e:
         logger.error("Research agent failed: %s", str(e), exc_info=True)
```

Ruff (0.8.2)

Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)